skills/csharp/experimental/eventskill/Responses/Shared/SharedResponses.zh-cn.lg

```diff
@@ -0,0 +1,58 @@
+[import](Shared.lg)
+[import](SharedResponsesTexts.zh-cn.lg)
+
+# DidntUnderstandMessage(Data, Cards, Layout)
+[Activity
+    Text = ${DidntUnderstandMessage.Text(Data)}
+    Speak = ${DidntUnderstandMessage.Text(Data)}
+    Attachments = ${if(Cards == null, null, foreach(Cards, Card, CreateCard(Card)))}
+    AttachmentLayout = ${if(Layout == null, 'list', Layout)}
+    InputHint = acceptingInput
+]
+
+# CancellingMessage(Data, Cards, Layout)
+[Activity
+    Text = ${CancellingMessage.Text(Data)}
+    Speak = ${CancellingMessage.Text(Data)}
+    Attachments = ${if(Cards == null, null, foreach(Cards, Card, CreateCard(Card)))}
+    AttachmentLayout = ${if(Layout == null, 'list', Layout)}
+    InputHint = acceptingInput
+]
+
+# NoAuth(Data, Cards, Layout)
+[Activity
+    Text = ${NoAuth.Text(Data)}
+    Speak = ${NoAuth.Text(Data)}
+    Attachments = ${if(Cards == null, null, foreach(Cards, Card, CreateCard(Card)))}
+    AttachmentLayout = ${if(Layout == null, 'list', Layout)}
+    InputHint = expectingInput
+]
+
+# AuthFailed(Data, Cards, Layout)
+[Activity
+    Text = ${AuthFailed.Text(Data)}
```

Comment:
this lg is different with other language.

Reply:
Good catch. changed other languages to zh-cn.lg